src/couch/src/couch_doc.erl

```diff
@@ -270,6 +270,9 @@ transfer_fields([{<<"_replication_state">>, _} = Field | Rest],
 transfer_fields([{<<"_replication_state_time">>, _} = Field | Rest],
     #doc{body=Fields} = Doc) ->
     transfer_fields(Rest, Doc#doc{body=[Field|Fields]});
+transfer_fields([{<<"_replication_start_time">>, _} = Field | Rest],
```

Comment:
Its the backwards replication that hurts yeah. I wanted to add a new _ field for quorum info once but it got shot down cause it'd break replication with every CouchDB instance older than when it was added.